lib/elixir/pages/getting-started/writing-documentation.md

```diff
@@ -1,18 +1,31 @@
-# Writing documentation
-
-Elixir treats documentation as a first-class citizen. Documentation must be easy to write and easy to read. In this guide you will learn how to write documentation in Elixir, covering constructs like module attributes, style practices, and doctests.
+# Writing Documentation
 
+Elixir elevates documentation to first-class status, emphasizing ease of writing as well as readability. This guide will teach you to craft effective documentation in Elixir, covering essential constructs like module attributes, typespecs, style practices, and doctests.
 ## Markdown
 
-Elixir documentation is written using Markdown. There are plenty of guides on Markdown online, we recommend the one from GitHub as a getting started point:
+Elixir documentation is written in Markdown. Some commonly used elements are: 
```

Comment:
We don't plan to add a small markdown summary here. Other resources do it better.